compactor2/src/components/target_level/mod.rs

```diff
@@ -0,0 +1,11 @@
+use std::fmt::{Debug, Display};
+
+use data_types::CompactionLevel;
+
+pub mod hot_cold;
+pub mod naive;
+
+pub trait TargetLevelDetection: Debug + Display + Send + Sync {
+    // return the compaction level the given files are suitable to get compacted to
```

Comment:
```suggestion
    /// return the compaction level the given files are suitable to get compacted to
```

Docstrings require `///` (in contrast to normal code comments).